utils/zerovec/src/ule/mod.rs

```diff
@@ -261,7 +261,7 @@ where
 /// Safety checklist for `VarULE`:
 ///
 /// 1. The type *must not* include any uninitialized or padding bytes.
-/// 2. The type must have an alignment of 1 byte.
+/// 2. The type must have an alignment of 1 byte, or it is a ZST or an empty enum.
```

Comment:
@Manishearth Flagging this code for a more detailed unsafe review.

Reply:
let's exclude empty enum

"or it is a ZST that is safe to construct" so that we disallow free construction of token ZSTs